### PR TITLE
generate sample mcts kids

### PR DESCRIPTION
**Pull request by frankhuster**
Fire up MOTECH.

Reset the db & create domain data and then create a file containing 1000 children, in your mim repo directory root:
```
python performance/data/data.py --cleardb --mctskids=1000
```
Import the generated file using MOTECH, and assuming tomcat7 is installed as a service:
```
curl -v -F "csvFile=@/var/lib/tomcat7/testing-directory/mctskids.csv" http://localhost:8080/motech-platform-server/module/kilkari/child/import
```

### CODE REVIEW

File: checkstyle.xml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.2//EN" "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
 <module name="Checker">
+    <module name="SuppressionFilter">
+        <property name="file" value="suppressions.xml"/>
```
Comment (larubbio) [inline checkstyle.xml]: This seems like a slippery slope..... ;)

File: kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java [hunk 2]
```diff
@@ -125,11 +126,11 @@ public int importMotherData(Reader reader) throws IOException {
                 importMotherRecord(record);
                 count++;
                 if (count % PROGRESS_INTERVAL == 0) {
-                    LOGGER.debug("Imported {}", timer.frequency(count));
+                    LOGGER.debug(IMPORTED, timer.frequency(count));
```
Comment (larubbio) [inline kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java]: Did the CPD make you do this?

Comment (frankhuster) [inline kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java]: Not sure if it was PMD or CPD. But the same constant was used 4 times and
it tripped an error, not just a warning.

On Mon, Sep 28, 2015 at 1:11 PM, Rob LaRubbio <notifications@github.com>
wrote:

> In
> kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java
> <https://github.com/motech-implementations/mim/pull/592#discussion_r40599161>
> :
>
> > @@ -125,11 +126,11 @@ public int importMotherData(Reader reader) throws IOException {
> >                  importMotherRecord(record);
> >                  count++;
> >                  if (count % PROGRESS_INTERVAL == 0) {
> > -                    LOGGER.debug("Imported {}", timer.frequency(count));
> > +                    LOGGER.debug(IMPORTED, timer.frequency(count));
>
> Did the CPD make you do this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/592/files#r40599161>.
>



-- 
Frank Huster
Mobile Health Innovations
Grameen Foundation
+1.206.325.6690 x201

**Comment by larubbio** [comment]
lgtm

**Pull request merged**